modules/select/js/select_directive.js

```diff
@@ -1261,6 +1268,34 @@
                 $element.find('.lx-select-selected__filter input').focus();
             });
         });
+
+        /**
+         * When the end of the dropdown is reached and infinite scroll is specified,
+         * fetch new data.
+         * 
+         * @param {Event}  evt        The scroll event.
+         * @param {string} dropdownId The id of the dropdown that scrolled to the end.
+         */
+        $scope.$on('lx-dropdown__scroll-end', async (evt, dropdownId) => {
```

Comment:
Lumx isn't transpiled by babel, you should avoid using `async/await` and arrow function